server/controllers/stock/core.js

```diff
@@ -496,6 +496,7 @@ function processMultipleLots(inventories) {
       lot.S_LOT_LIFETIME = lot.lifetime - lotLifetime;
       lot.S_RISK = lot.S_LOT_LIFETIME - lot.S_MONTH;
       lot.S_RISK_QUANTITY = lot.S_RISK * lot.avg_consumption;
+      lot.S_RISK_QUANTITY = Math.round(lot.S_RISK_QUANTITY);
```

Comment:
It's slightly confusing to set `lot.S_RISK_QUANTITY` twice.  Could you just wrap the first line with`Math.round()`?